File: app/src/main/java/com/example/ryne/myapplication/Java/entity/response/Taxon.java
package com.example.ryne.myapplication.Java.entity.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Taxon {
    @SerializedName("id")
    @Expose
    private String id;

    @SerializedName("name")
    @Expose
    private String name;

    @SerializedName("pretty_name")
    @Expose
    private String prettyName;

    @SerializedName("permalink")
    @Expose
    private String permalink;

    @SerializedName("parent_id")
    @Expose
    private String parentId;

    @SerializedName("taxonomy_id")
    @Expose
    private String taxonomyId;

    @SerializedName("meta_title")
    @Expose
    private String metaTitle;

    @SerializedName("meta_description")
    @Expose
    private String metaDescription;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPrettyName() {
        return prettyName;
    }

    public void setPrettyName(String prettyName) {
        this.prettyName = prettyName;
    }

    public String getPermalink() {
        return permalink;
    }

    public void setPermalink(String permalink) {
        this.permalink = permalink;
    }

    public String getParentId() {
        return parentId;
    }

    public void setParentId(String parentId) {
        this.parentId = parentId;
    }

    public String getTaxonomyId() {
        return taxonomyId;
    }

    public void setTaxonomyId(String taxonomyId) {
        this.taxonomyId = taxonomyId;
    }

    public String getMetaTitle() {
        return metaTitle;
    }

    public void setMetaTitle(String metaTitle) {
        this.metaTitle = metaTitle;
    }

    public String getMetaDescription() {
        return metaDescription;
    }

    public void setMetaDescription(String metaDescription) {
        this.metaDescription = metaDescription;
    }

    @Override
    public String toString() {
        return this.prettyName;
    }
}

File: app/src/main/java/com/example/ryne/myapplication/Kotlin/entity/response/Taxon.kt
package com.example.ryne.myapplication.Kotlin.entity.response

import com.google.gson.annotations.SerializedName

class Taxon(
        @SerializedName("id")

        var id: String? = null,

        @SerializedName("name")

        var name: String? = null,

        @SerializedName("pretty_name")

        var prettyName: String? = null,

        @SerializedName("permalink")

        var permalink: String? = null,

        @SerializedName("parent_id")

        var parentId: String? = null,

        @SerializedName("taxonomy_id")

        var taxonomyId: String? = null,

        @SerializedName("meta_title")

        var metaTitle: String? = null,

        @SerializedName("meta_description")

        var metaDescription: String? = null
) {
    override fun toString(): String {
        return this.prettyName.toString()
    }
}

